templates/group_vars_mash_servers

```diff
@@ -1932,6 +1937,43 @@ echoip_container_labels_traefik_tls_certResolver: "{{ devture_traefik_certResolv
 # /role-specific:echoip
 
 
+# role-specific:endlessh
+########################################################################
+#                                                                      #
+# endlessh                                                             #
+#                                                                      #
+########################################################################
+
+endlessh_enabled: false
+endlessh_hostname: "{{ mash_playbook_metrics_exposure_hostname }}"
```

Comment:
Hmm.. `endlessh_hostname` is used for the endlessh web UI, not just for metrics exposure, right.

This seems like an unfortunate default.

Because of it: people who don't define `endlessh_hostname` (thinking it won't expose the web UI), but use `mash_playbook_metrics_exposure_hostname` (for other reasons) will be surprised by the fact that the endlessh web UI is actually exposed on the "global metrics hostname". Not only that, but since they never intended to expose its web UI, they likely would not have configured Basic Authentication (`endlessh_container_labels_middleware_basic_auth_*`) and will have the web UI served publicly.

-----

I think that:

- `endlessh_hostname` and `endlessh_path_prefix` should not have default values

- the role should get another set of `_traefik` variables for handling metrics and their exposure (see [the group vars for promtail](https://github.com/mother-of-all-self-hosting/mash-playbook/blob/3c18123de9ab76139e4ef7539bf58dfef79cabc6/templates/group_vars_mash_servers#L4710-L4720) for an example)

This way, the defaults are sane and secure.. And metrics may be exposed on a completely different hostname and path prefix and with completely different Basic Auth credentials (auto-wired via `mash_playbook_metrics_exposure_http_basic_auth_enabled` and `mash_playbook_metrics_exposure_http_basic_auth_users`).